_drafts/oryx/browser-compatibility.md

```diff
@@ -0,0 +1,24 @@
+# Supported browsers / viewports
+Oryx supports and is being tested on all modern browsers and operating systems:
+
+* iOS 16.x (Safari latest, Google Chrome latest)
+* Android 13.x (Google Chrome latest)
+* macOS Ventura (Safari latest, Google Chrome latest) 
+* Windows 10 and higher (Google Chrome latest, Edge latest)
```

Comment:
Maybe let's add a note that it work's on Linux systems as well, but we are not testing just not testing it?

Reply:
Linux is kinda tricky - it does not have a specific version or number, and listing all distributives here will be a mess 
like `64-bit Ubuntu 18.04+, Debian 10+, openSUSE 15.2+, or Fedora Linux 32+`

Reply:
I'd better not add it because Linux is used by a very small number of e-commerce customers :)

Reply:
or maybe someone can suggest how it can be added without this list of different distributives

Reply:
added